### PR TITLE
Add Persistent Volumes Usage Grafana dashboard

### PR DESCRIPTION
**Pull request by desaintmartin**

### CODE REVIEW

**Comment by metalmatze** [comment]
Hey! Thanks for you contribution. Can you provide the Dashboard with a screenshot (it's basically a UI).
I'm not able to test this dashboard, as I don't have any `kubelet_volume` metrics on my cluster. Maybe @tomwilkie or @brancz can give it a try?

**Comment by desaintmartin** [comment]
Sure, here you are: 
Screenshot

File: dashboards/persistentvolumesusage.libsonnet [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      ).addTarget(prometheus.target(
+        |||
+          kubelet_volume_stats_inodes_used{%(kubeletSelector)s, persistentvolumeclaim="$volume"} / kubelet_volume_stats_inodes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} * 100
+        ||| % $._config,
+        legendFormat='{{ Usage }}',
+        intervalFactor=1,
+      ));
+
+      dashboard.new(
+        'Kubernetes Persistent Volumes usage',
```
Comment (metalmatze) [inline dashboards/persistentvolumesusage.libsonnet]: We currently have dashboards called `Node`, `Pods` and so on, or we prefix them with `K8s / ...`. We should be consistent and call this one `Persistent Volumes`.

File: dashboards/persistentvolumesusage.libsonnet [hunk 2]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (metalmatze) [inline dashboards/persistentvolumesusage.libsonnet]: We don't need this first blank line.

File: dashboards/persistentvolumesusage.libsonnet [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+local graphPanel = grafana.graphPanel;
+local promgrafonnet = import '../lib/promgrafonnet/promgrafonnet.libsonnet';
+local numbersinglestat = promgrafonnet.numbersinglestat;
+local gauge = promgrafonnet.gauge;
+
+{
+  grafanaDashboards+:: {
+    'persistentvolumesusage.json':
+      local sizeGraph = graphPanel.new(
+        'Volume size usage',
```
Comment (metalmatze) [inline dashboards/persistentvolumesusage.libsonnet]:
```suggestion
        'Volume Space Usage',
```
To be consistent with the Nodes dashboard.

File: dashboards/persistentvolumesusage.libsonnet [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      ).addTarget(prometheus.target(
+        |||
+          (kubelet_volume_stats_capacity_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} - kubelet_volume_stats_available_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"}) / kubelet_volume_stats_capacity_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} * 100
+        ||| % $._config,
+        legendFormat='{{ Usage }}',
+        intervalFactor=1,
+      ));
+
+      local inodesGraph = graphPanel.new(
+        'Volume inodes usage',
```
Comment (metalmatze) [inline dashboards/persistentvolumesusage.libsonnet]:
```suggestion
        'Volume inodes Usage',
```

**Comment by desaintmartin** [comment]
Done!

**Comment by metalmatze** [comment]
Awesome!

**Pull request merged**